Jellyfin.Api/Controllers/ImageController.cs

```diff
@@ -94,7 +93,53 @@ public class ImageController : BaseJellyfinApiController
         public async Task<ActionResult> PostUserImage(
             [FromRoute, Required] Guid userId,
             [FromRoute, Required] ImageType imageType,
-            [FromRoute] int? index = null)
+            [FromQuery] int? index = null)
+        {
+            if (!RequestHelpers.AssertCanUpdateUser(_authContext, HttpContext.Request, userId, true))
+            {
+                return Forbid("User is not allowed to update the image.");
+            }
+
+            var user = _userManager.GetUserById(userId);
+            await using var memoryStream = await GetMemoryStream(Request.Body).ConfigureAwait(false);
+
+            // Handle image/png; charset=utf-8
+            var mimeType = Request.ContentType.Split(';').FirstOrDefault();
```

Comment:
Is this the only mime type that will be received, or should this be...

```suggestion
            var mimeType = Request.ContentType.Split(';')?.FirstOrDefault() ?? Request.ContentType;
```

Reply:
This is a relic of the service stack api. This PR doesn’t add any new functionality, just splits controllers that had multiple routes